### ISSUE TITLE
How "server add"? 

### ISSUE THREAD

**Issue by csoyuncu** [CS2Browser/cs2-server-browser#50]
hello. how can i add my cs2 server ip to cs2browser web site?

**Comment by MDTH0** [comment]
It should just show up on the site as long as your game server login token is setup correctly

**Comment by csoyuncu** [comment]
92.60.70.20:27015 my server ip. i cant see ?

**Comment by JMorell** [comment]
Hi, is the server up and do you have a token? It doesn't appear in the Steam API

**Comment by csoyuncu** [comment]
yes, server has a token. and i can play at this server. Can you test "connect 92.60.70.20"

**Comment by JMorell** [comment]
When I try to get the information from the Steam API, nothing is returned.

(you need a valid API KEY)
https://api.steampowered.com/IGameServersService/GetServerList/v1/?key=API_KEY&filter=\addr\92.60.70.20:27015

**Comment by csoyuncu** [comment]
> When I try to get the information from the Steam API, nothing is returned.
> 
> (you need a valid API KEY) [https://api.steampowered.com/IGameServersService/GetServerList/v1/?key=API_KEY&filter=\addr\92.60.70.20:27015](https://api.steampowered.com/IGameServersService/GetServerList/v1/?key=API_KEY&filter=%5Caddr%5C92.60.70.20:27015)

can you try 92.60.70.1:27016

**Comment by JMorell** [comment]
![image](https://github.com/user-attachments/assets/9ab110ad-4968-477b-9a15-4ed1be689e46)

92.60.70.1:27015 works.

**Comment by csoyuncu** [comment]
> ![image](https://private-user-images.githubusercontent.com/6134888/358539171-9ab110ad-4968-477b-9a15-4ed1be689e46.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4NDA0NDcsIm5iZiI6MTcyMzg0MDE0NywicGF0aCI6Ii82MTM0ODg4LzM1ODUzOTE3MS05YWIxMTBhZC00OTY4LTQ3N2ItOWExNS00ZWQxYmU2ODllNDYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTZUMjAyOTA3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzIwYTdiNmJmNDMxNzFmNjJhMDBiODU3YmViZWJiYjBhOWJlMGFhODRhODllZmJlMzM5M2FlNjQ5NzhiMjA3NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.5-KeDL82XfJwQmxiwjXt6F5css2eNkYg71eAJhZZvMk)
> 
> 92.60.70.1:27015 works.

port is 27016, not 27015 (there is cs 1.6 server at 27015 port)

**Comment by JMorell** [comment]
in 27016 nothing is found in the official api

**Comment by csoyuncu** [comment]
> in 27016 nothing is found in the official api

{
  "response": {
    "servers": [
      {
        "addr": "92.60.70.1:27016",
        "gameport": 27016,
        "steamid": "85568392932001634",
        "name": "CSOYUNCU.COM CS2 PRO PUBL!C SERVER !ws !knife !gloves | CS2TURK",
        "appid": 730,
        "gamedir": "csgo",
        "version": "1.40.2.6",
        "product": "cs2",
        "region": -1,
        "players": 0,
        "max_players": 36,
        "bots": 7,
        "map": "de_dust2",
        "secure": true,
        "dedicated": true,
        "os": "l",
        "gametype": "empty,cs2,cs2turkiye.com,csoyuncu,server,secure"
      }
    ]
  }
}

**Comment by JMorell** [comment]
this one is: https://cs2browser.com/gameserver/181703/csoyuncu-com-cs2-pro-publ-c-server-ws-knife-gloves-cs2turk/

**Comment by crashzk** [comment]
For the record, only one of my two servers is online on the site as well. I checked both in the API and everything returned correctly in the query.

Server IPS:
- 45.179.91.41:27176
- 45.179.91.41:27156

Just wait to see if it updates and adds, or would it be an error? Servers have been online for months now, without changing anything.

**Comment by JMorell** [comment]
If the server is empty for more than 24 hours, the server does not appear.
